src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RisksAndNeedsService.kt

```diff
@@ -22,7 +22,7 @@ data class CreateSupplementaryRiskRequest(
   val crn: String,
   val createdByUser: String,
   val createdByUserType: String,
-  val createdDate: LocalDateTime,
+  val createdDate: LocalDateTime?,
```

Comment:
It sounds like this field shouldn't be optional -- intent-wise there must always be a time where the risk is 'signalled'

What do you think about setting `additionalRiskInformationUpdatedAt` to current time in `ReferralService.submitAdditionalRiskInformation` if it's missing? (Together with a warning?)

Reply:
yes happy with that idea.